docs/assets/templates/tools/.dockstore.yml

```diff
@@ -0,0 +1,60 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of tools. Each element corresponds to a tool entry on Dockstore.
+tools:
+  
+  # The optional name for a tool, which may only consist of alphanumerics
+  # and internal underscores and hyphens. Names may not exceed 256 characters. 
+  # If using a ``/.dockstore.yml`` with multiple tools, this field is required 
+  # to uniquely identify tools in the repository. Otherwise, it is optional.
+  # 
+  # It should be noted that having the name come first is an arbitrary decision.
+  # You could use subclass instead, for instance. Provided arrays are not broken
+  # up, the order of fields within a .dockstore.yml is not important.
+  - name: <String>
+    
+    # The descriptor language used for the tool. As a reminder, we generally advise against WDL tools.
+    # Only select one of these -- if your tool has a CWL and a WDL version, then register them each
+    # as a seperate entry.
+    subclass: <CWL | WDL>
+    
+    # Tool-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.
+    # Set to true to publish an unpublished tool, or false to unpublish a published tool.
+    # Omitting the publish setting leaves the publish-state unchanged (recommended for all non-primary branches).
+    publish: <Boolean>
+
+    # The absolute path to the primary descriptor file in the Git repository.
+    # - For CWL, the primary descriptor is a .cwl file.
+    # - For WDL, the primary descriptor is a .wdl file.
+    primaryDescriptorPath: <String>
+
+    # An optional array of absolute paths to test parameter files in the Git repository.
+    # For example...
+    # testParameterFiles:
+    #     - /null-model/null-model.json
+    #     - /null-model/null-model-binary.json
+    testParameterFiles: <String Array>
+
+    # An optional array of authorship information, requiring at least the ``name`` of each author.
```

Comment:
Should be just the `orcid` actually 
https://github.com/dockstore/dockstore-documentation/pull/178 for details